python/nav/util.py

```diff
@@ -36,6 +36,11 @@
     from importlib_resources import files, as_file
 
 
+# rename to a better name
+resource_files = files
+del files
+
+
```

Comment:
I would have thought `from importlib_resources import files as resource_files` would be more legible... :-/

Reply:
Is that combinable with multiple imports on one line? Hm.